routes/index.tsx

```diff
@@ -33,23 +34,28 @@ export function compareScore(a: Item, b: Item) {
 }
 
 export const handler: Handlers<HomePageData, State> = {
-  async GET(_req, ctx) {
+  async GET(req, ctx) {
     /** @todo Add pagination functionality */
-    const items = (await getAllItems({ limit: 10 })).sort(compareScore);
+    const { searchParams } = new URL(req.url);
+    const limit = Math.min(30, ~~(searchParams.get("limit") || 10));
+    const start = decodeURI(searchParams.get("page") || '');
+    const { items: itemsRaw, cursor } = await getAllItems({ limit, cursor: start });
+    const items = itemsRaw.sort(compareScore);
```

Comment:
```suggestion
    const start = new URL(req.url).searchParams.get("page") || undefined;
    const { items, cursor } = await getAllItems({ limit: 10, cursor: start });
    items.sort(compareScore);
```
- The `start` constant can be calculated in one readable line.
- A hardcoded limit of 10 is fine.
- `Array.sort()` mutates the original array. So no need to create another array.